pkg/controllers/csi/driver/volumes/host/publisher.go

```diff
@@ -18,30 +18,28 @@ package hostvolumes
 
 import (
 	"context"
-	goerrors "errors"
 	"fmt"
 	"os"
+	"time"
 
 	csivolumes "github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi/driver/volumes"
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi/metadata"
 	"github.com/container-storage-interface/spec/lib/go/csi"
-	"github.com/pkg/errors"
 	"github.com/spf13/afero"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"gorm.io/gorm"
-	"k8s.io/utils/mount"
+	"k8s.io/mount-utils"
 )
 
-const failedToGetOsAgentVolumePrefix = "failed to get OSMount from database: "
+const failedToGetOsAgentVolumePrefix = "failed to get osagent volume info from database: "
 
 func NewHostVolumePublisher(fs afero.Afero, mounter mount.Interface, db metadata.Access, path metadata.PathResolver) csivolumes.Publisher {
 	return &HostVolumePublisher{
 		fs:           fs,
 		mounter:      mounter,
 		db:           db,
 		path:         path,
-		isNotMounted: mount.IsNotMountPoint,
+		isNotMounted: mount.IsNotMountPoint, //nolint
```

Comment:
could be just removed, not used in actual code

Reply:
fixed in 994eafc2